tasks/task_2.py

```diff
@@ -0,0 +1,43 @@
+from selenium import webdriver
+from bs4 import BeautifulSoup
+import time
+import pandas as pd
+
+url = "https://www.youtube.com/@zusmani78/videos"
+driver = webdriver.Chrome(executable_path="chromedriver")
+driver.get(url)
+last_height = driver.execute_script("return document.documentElement.scrollHeight")
+
+while True:
+    driver.execute_script("window.scrollTo(0, arguments[0]);", last_height)
+    time.sleep(1.5)
+    new_height = driver.execute_script("return document.documentElement.scrollHeight")
+    if new_height == last_height:
+        break
+    last_height = new_height
```

Comment:
Can we create a generic function out of this `def scroll_to_page_end(driver)` which will take driver as parameter and scroll the opened page to the end of page?

Move this function to a `utils.py` file, and import, and call that here.

Reply:
ok